03/app.js

```diff
@@ -1,18 +1,108 @@
-const min = 1;
-const max = 100;
+let parsedNumbers = [];
+let numbersSum = 0;
 
-const a = randomNumber(min, max);
-const b = randomNumber(min, max);
-const c = randomNumber(min, max);
 
-console.log(a, b, c);
+const a = 5;
+const b = 40;
+const c = 25;
 
+
+const getSum = function(a,b,c) {
+  
+  parsedNumbers.push(a,b,c)
+  
+  parsedNumbers.sort(function(a,b) {
+    return b - a;
+  });
+  
+  const highestNumbers = parsedNumbers.slice(0,2)
+  
+  highestNumbers.forEach(function(item) {
+    numbersSum += item
+    });
+  
+  return numbersSum
+  
+}
+
+const sum = getSum(a,b,c)
+console.log(sum)
+  
+
+const isEven = function(num) {
+  if(typeof num != "number") {
+    return null
+  }
+  else if (typeof num == 'number') {
```

Comment:
Nie trzeba drugi raz sprawdzać, jeśli wyżej mamy, że jest różne od `number` to odwrotność tego to że jest `number`.
My już to wiemy z poprzedniego warunku więc nie musimy tego sprawdzać :)